PhysicsTools/Heppy/python/analyzers/objects/JetAnalyzer.py

```diff
@@ -387,15 +387,16 @@ def smearJets(self, event, jets):
             if gen != None:
                genpt, jetpt, aeta = gen.pt(), jet.pt(), abs(jet.eta())
                # from https://twiki.cern.ch/twiki/bin/view/CMS/JetResolution
-               #8 TeV tables
+               #13 TeV tables from
+               #https://twiki.cern.ch/twiki/bin/view/CMS/JetResolution#Smearing_procedures
                
-               factor = 1.079 + self.shiftJER*0.026
-               if   aeta > 3.2: factor = 1.056 + self.shiftJER * 0.191
-               elif aeta > 2.8: factor = 1.395 + self.shiftJER * 0.063
-               elif aeta > 2.3: factor = 1.254 + self.shiftJER * 0.062
-               elif aeta > 1.7: factor = 1.208 + self.shiftJER * 0.046
-               elif aeta > 1.1: factor = 1.121 + self.shiftJER * 0.029
-               elif aeta > 0.5: factor = 1.099 + self.shiftJER * 0.028
+               factor = 1.061 + self.shiftJER*0.023
```

Comment:
@aelwood.
there is a conflict on this with the HEAD after the #600 me, move the 13TeV parameters in the function shiftJERfactor
https://github.com/CERN-PH-CMG/cmg-cmssw/pull/600/files#diff-b4f7bda6e988f3a66e98f9d2f807d14cR51